build.gradle

```diff
@@ -28,6 +28,7 @@ apply from: rootProject.projectDir.path + '/gradle/scripts/jacoco-coveralls-supp
 
 idea.project {
   ext.languageLevel = JavaVersion.VERSION_1_7
```

Comment:
Chavdar mentioned at some point that was necessary for some new version of Gradle, so it is there in preparation for the migration.